certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -234,91 +234,106 @@ func (p *DeployableByOlmCheck) grantRegistryPermissionToServiceAccount(ctx conte
 	return nil
 }
 
+type watchFunc func(ctx context.Context, engine cli.OpenshiftEngine, name, namespace string) (string, bool, error)
```

Comment:
Are we using the `timeout time.Duration`? Back to the comment that I left earlier, I think line 240 should be `ctx, cancel := context.WithTimeout(ctx, timeout)`.

Reply:
Uh I meant to tag line 239. My bad :(

Reply:
Oh... THAT'S what you were saying. I thought you were talking about the actual function order. Not the WithTimeout call... Let me look at that.

Reply:
I have to do a better job at explaining my point. Thank you for the fix!